eclair-core/src/main/scala/fr/acinq/eclair/crypto/ChaCha20Poly1305.scala

```diff
@@ -94,6 +96,8 @@ object ChaCha20 {
   */
 object ChaCha20Poly1305 extends Logging {
 
+  val keyLogger = Logger("keylog")
```

Comment:
How about adding a small comment here? In some time, it will be hard to remember why we've put this here.

Reply:
Sure, good idea.